low-level/files/opt/seagate/sspl/setup/constants.sh

```diff
@@ -15,9 +15,9 @@
 
 # This script contains common constants used by various provisioner scripts
 
-PRODUCT_NAME='LDR_R2'
+PRODUCT_NAME='LR2'
```

Comment:
Why changed to LR2, we need to align with standard name being used in whole Cortx, dont use what is convenient.